### DISCUSSION TITLE
Improved `diff` code block rendering

### DISCUSSION THREAD

**Discussion by kylebutts** [quarto-dev/quarto-cli#8387] [Q&A] [answered]
### Description

A diff block in github highlights changes using standard red and green line highlighting:
```diff
- a <- 14
+ a <- 12
```

In quarto, it renders the diff in a weird way (delete = green, + = nothing). Is this something that could easily be improved?
`````
---
format: html
---

```diff
- a <- 14
+ a <- 12
```
`````
![quarto diff example](https://github.com/quarto-dev/quarto-cli/assets/19961439/8c7dc2c2-95cd-430d-9399-b9406b8ec5a1)

**Comment by mcanouil** [comment, marked as answer]
This depends on the highlighting style which is provided by Pandoc or more accurately Skylighting.
If you are not happy with the style, change it, see <https://quarto.org/docs/output-formats/html-code.html#highlighting>.

(I changed this to Q&A, because, there is actually nothing to be done)

<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/9f73fd56-1f3d-4b5b-a0e9-bd3b499d4858">

````qmd
---
title: "Quarto Playground"
format: html
highlight-style: github
---

This is a playground for Quarto.

```diff
- a <- 14
+ a <- 12
```
````
